README.md

```diff
@@ -28,6 +28,17 @@ The fastest way to experience the operator is to follow the [Quick start guide](
 + However, this release candidate is suitable for testing and early adopters.
 ```
 
+## Known issues
+
+| Issue | Description |
+| --- | --- |
+| [#721](https://github.com/oracle/weblogic-kubernetes-operator/issues/721) | Incorrect default domainHome when domainHomeInImage is true |
```

Comment:
Incorrect default domainHome when domainHomeInImage is true -> Incorrect default `domainHome` when `domainHomeInImage` is true.

Add a period to the end of the following sentence also.